tempodb/encoding/vparquet3/schema.go

```diff
@@ -221,11 +227,12 @@ type Trace struct {
 	TraceIDText string `parquet:",snappy"`
 
 	// Trace-level attributes for searching
-	StartTimeUnixNano uint64 `parquet:",delta"`
-	EndTimeUnixNano   uint64 `parquet:",delta"`
-	DurationNano      uint64 `parquet:",delta"`
-	RootServiceName   string `parquet:",dict"`
-	RootSpanName      string `parquet:",dict"`
+	StartTimeUnixNano uint64        `parquet:",delta"`
+	EndTimeUnixNano   uint64        `parquet:",delta"`
+	DurationNano      uint64        `parquet:",delta"`
+	RootServiceName   string        `parquet:",dict"`
+	RootSpanName      string        `parquet:",dict"`
+	ServicesMeta      []ServiceMeta `parquet:",list"`
```

Comment:
the parquet-go library will technically attempt schema changes automatically but we have found it to be costly and, at times, unstable.

if we want to add this field i think we should cut a vparquet4. wdyt @mdisibio ?

Reply:
Yes this was mentioned in the PR description above - we're on the same page and this is just a proof-of-concept.

Reply:
Is there a branch for the next version of the parquet schema?